apps/ws/src/socket/services/web-socket.worker.ts

```diff
@@ -37,14 +39,16 @@ export class WebSocketWorker extends WebSocketsWorkerService implements INovuWor
           'WS Service',
           function () {
             const transaction = nr.getTransaction();
+            const { data: jobData } = job;
+            const data = jobData as IWebSocketDataDto;
```

Comment:
I am not sure what is the type of `job` above therefore i create this mapping so we could at least get the `data` object right here.